framework/arcane-framework/src/main/scala/services/mssql/MsSqlConnection.scala

```diff
@@ -166,6 +156,31 @@ object MsSqlConnection:
    */
   def apply(connectionOptions: ConnectionOptions): MsSqlConnection = new MsSqlConnection(connectionOptions)
 
+  /**
+   * Converts a SQL type to an Arcane type.
+   *
+   * @param sqlType The SQL type.
+   * @return The Arcane type.
+   */
+  def toArcaneType(sqlType: Int): ArcaneType = sqlType match
```

Comment:
match non-mapped case and maybe return option here

Reply:
I will use `Try` here because it allows to add additional context for error.